app/menus/otherMenu.js

```diff
@@ -18,8 +19,8 @@ const openAboutWindow = async (mainWindow) => {
     width: 300,
   });
 
-  await modal.loadFile('../app/menus/otherHelp.html', {
-    query: { version: process.env.npm_package_version },
+  await modal.loadFile('../../app/menus/otherHelp.html', {
```

Comment:
is this just how you have to do modals in electron? We can't use something like MUI's `Dialog` component? feels weird to have to load a whole other file rather than use a component with a z-index or something

Reply:
The issue here is that it isn't getting rendered as part of the app's view. It's a separate electron window that gets loaded, and apparently that requires it to be handled differently.